src/Integration/Monolog/CloudWatch/composer.json

```diff
@@ -0,0 +1,32 @@
+{
+    "name": "async-aws/monolog-cloudwatch",
+    "type": "library",
+    "description": "Monolog CloudWatch Logs adapter",
+    "keywords": [
+        "aws",
+        "Monolog",
+        "CloudWatch",
+        "Logs"
+    ],
+    "license": "MIT",
+    "require": {
+        "php": "^7.2.5",
+        "async-aws/cloud-watch-logs": "0.1.x-dev",
```

Comment:
`^0.1` should work

Reply:
async-aws/cloud-watch-logs is not released yet

```
  Problem 1
    - The requested package async-aws/cloud-watch-logs ^0.1 is satisfiable by async-aws/cloud-watch-logs[0.1.x-dev] but these conflict with your requirements or minimum-stability.
```

Reply:
It works better on CI ;)

There is some rules that allows each package do install dev-master if needed.

Reply:
same issue on CI

Reply:
I reverted it to ^0.1-dev, please let me know if I need to change it

Reply:
We can fix that before we tag a new version